src/NuGet.Services.BasicSearch/NuGet.Services.BasicSearch.csproj

```diff
@@ -130,9 +130,6 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.ApplicationInsights">
```

Comment:
Transitive dependency of `Microsoft.ApplicationInsights.Web`. Upgraded to version 2.10.0 as a result.